src/App.tsx

```diff
@@ -1,24 +1,242 @@
-/* eslint-disable max-len */
-/* eslint-disable jsx-a11y/control-has-associated-label */
-import React from 'react';
+import React, { useEffect, useMemo, useState } from 'react';
+import classNames from 'classnames';
 import { UserWarning } from './UserWarning';
+import {
+  deleteTodo,
+  getTodos,
+  patchTodo,
+  postTodo,
+} from './api/todos';
+import { Todo } from './types/Todo';
+import { TodoList } from './components/TodoList';
+import { FilterType } from './types/FilterType';
+import { TodoFilter } from './components/TodoFilter';
+import { Loader } from './components/Loader';
+import { Error } from './components/Error';
+import { TodoForm } from './components/TodoForm';
 
-const USER_ID = 0;
+const USER_ID = 6925;
 
 export const App: React.FC = () => {
+  const [errorMessage, setErrorMesage] = useState('');
+  const [hasError, setHasError] = useState(false);
+  const [todos, setTodos] = useState<Todo[]>([]);
+  const [isLoading, setIsloading] = useState(false);
```

Comment:
Typo in the state variable name: `setIsloading`. It should be `setIsLoading`. Please update the name to follow camelCase convention.